server/src/main/java/org/elasticsearch/search/aggregations/timeseries/TimeSeriesIndexSearcher.java

```diff
@@ -93,7 +93,7 @@ protected boolean lessThan(LeafWalker a, LeafWalker b) {
             this.collector.setScorer(scorer);
             iterator = scorer.iterator();
             docBase = context.docBase;
-            tsids = context.reader().getSortedSetDocValues(TimeSeriesIdFieldMapper.NAME);
+            tsids = context.reader().getSortedDocValues(TimeSeriesIdFieldMapper.NAME);
```

Comment:
I'd use `DocValues.getSorted(context.reader(), TimeSeriesIdFieldMapper.NAME)` here as it will protect you from NPEs and also automatically convert between SortedSet and Sorted if we need it.